transports/noise/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.40.1
+
+ - Changing `payload.proto` syntax to be proto2. See [PR 3007]
```

Comment:
This markdown link reference is missing its actual link.

```suggestion
 - Changing `payload.proto` syntax to be proto2. See [PR 3007].
 
[PR 3007]: https://github.com/libp2p/rust-libp2p/pull/3007
```

Reply:
Ahh, thanks!